etna/datasets/internal_datasets.py

```diff
@@ -275,4 +375,24 @@ def get_m4_dataset(dataset_dir: Path, dataset_freq: str) -> None:
         "freq": "D",
         "parts": ("train", "test", "full"),
     },
+    "m3_monthly": {
+        "get_dataset_function": partial(get_m3_dataset, dataset_freq="monthly"),
+        "freq": "M",
+        "parts": ("train", "test", "full"),
+    },
+    "m3_quarterly": {
+        "get_dataset_function": partial(get_m3_dataset, dataset_freq="quarterly"),
+        "freq": "Q-DEC",
+        "parts": ("train", "test", "full"),
+    },
+    "m3_yearly": {
+        "get_dataset_function": partial(get_m3_dataset, dataset_freq="yearly"),
+        "freq": "A-DEC",
+        "parts": ("train", "test", "full"),
+    },
+    "m3_other": {
```

Comment:
How is it different from `m3_quarterly`?

Reply:
If timestamps are unknown why are we using freq="Q-DEC"? Is it really quarterly data?

Reply:
in another library it is marked as quarterly data. actually we don't know. we will write about this in docs in future